bin/import-packaged-data-to-local.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# IMPORTANT: RUN THIS FROM THE ROOT DIRECTORY ideally as `just import-packaged-data-to-local`
+
+set -xeuo pipefail
+
+# TODO: handle running from the non-ideal working dir
+
+ZIP_FILE_PATH=$1
+
+if [[ -z $ZIP_FILE_PATH ]];
```

Comment:
I think `[[ ! -f "$ZIP_FILE_PATH" ]]` would be better as that also check to see if the given path is a regular file.